pmm-app/src/pmm-qan/panel/components/Details/database-models/mysql/mysql.ts

```diff
@@ -44,11 +44,11 @@ export const mysqlMethods = {
     return result.mysql_show_table_status.action_id;
   },
 
-  getExplainJSON: async ({ example, queryId, placeholders }) => {
+  getExplainJSONMysql: async ({ example, queryId, placeholders }) => {
     try {
       const payload = getExplainPayload(example, queryId, placeholders);
 
-      const result = await MysqlDatabaseService.getTraditionalExplainJSONMysql(payload);
+      const result = await MysqlDatabaseService.getExplainJSONMysql(payload);
```

Comment:
Same as with the `mysqlmethods`, the mysql in the method name is redundant.

Reply:
Updated.